lms/static/scripts/frontend_apps/components/SubmitGradeForm.tsx

```diff
@@ -207,24 +210,16 @@ export default function SubmitGradeForm({
 
   return (
     <>
-      <form autoComplete="off">
+      <form autoComplete="off" onSubmit={onSubmitGrade} ref={formRef}>
         <label htmlFor={gradeId} className="font-semibold text-xs">
           Grade (Out of {scoreMaximum})
         </label>
         <div className="flex">
           <span className="relative w-14">
-            {validationMessage && (
-              <ValidationMessage
-                message={validationMessage}
-                open={showValidationError}
-                onClose={() => {
-                  // Sync up the state when the ValidationMessage is closed
-                  setValidationError(false);
-                }}
-              />
-            )}
             <Input
               classes={classnames(
+                // Hide up/down arrow buttons (https://stackoverflow.com/a/75872055)
+                '[appearance:textfield] [&::-webkit-outer-spin-button]:appearance-none [&::-webkit-inner-spin-button]:appearance-none',
```

Comment:
@robertknight One decimal place seems reasonable for the improved grades where we use the actual assignment point value, but it's a change from how the old version works (as far as I'm aware). I've seen instructors in the old grading version I've seen professors use more decimal places because they were trying to work with the "out of 10" point system. So if they had an assignment worth three points some instructors would would give 3.33 out of 10, etc.

It's always possible we'll find someone wanting to use more decimal points. We can add this to our documentation as needed. If possible it might be helpful to make sure there's an informative error message if someone **does** use more than one decimal place.

Reply:
> So if they had an assignment worth three points some instructors would give 3.33 out of 10, etc.

We've now got the ability to reflect the actual number of points configured in the LMS, so this should address this need in a much more sane way, although I'm aware we'll have LMSes using LTI 1.1 where we don't have the ability to read the grade.

We can adjust this minimum increment if needed. @acelaya we can also remove the minimum increment entirely by using `step="any"`.

> If possible it might be helpful to make sure there's an informative error message if someone does use more than one decimal place.

Yes, the browser will tell the user if they try to enter a value with too many decimal places. If a user enters "3.33333" the browser will tell them something like "The nearest allowed values are 3.2 and 3.3".

Reply:
I think we should keep the services as closer to how it currently works as possible. I'm going to check exactly how many decimal points we allow at the moment, and try combinations of the `step` attribute that allow that.

Reply:
What I can see is that we currently allow any amount of decimals, but then we round it to two decimals when loading the grade again.

I'm going to try and allow two decimals for now.

Reply:
I'll use `step="any"` as suggested by Rob, as it is what has a behavior closer to what we allow now.